pysb/tests/test_util.py

```diff
@@ -0,0 +1,41 @@
+from pysb.core import *
```

Comment:
I think we should move to eliminate "import *" from our own code eventually but I've written enough of those in this codebase that I'm not going to fault you for this one. :) Pass!